### PR TITLE
get-dc.sh -> install_custom_dc.sh , make messages more clear

### PR DESCRIPTION
**Pull request by Fructokinase**
Here's an example of how it will look.

image

### CODE REVIEW

File: scripts/install_custom_dc.sh [hunk 1]
```diff
@@ -83,8 +83,29 @@ terraform init && terraform apply \
...
-echo "You should expect the instance to be accessible via https://$PROJECT_ID-datacommons.com"
-echo "within 30 minutes or so after custom-datacommons-support@google.com responds."
+_success_msg="
+###############################################################################
+# Status: Successfully launched the installer in $PROJECT_ID.
+###############################################################################
+
+###############################################################################
+# Action required:
+###############################################################################
```
Comment (shifucun) [inline scripts/install_custom_dc.sh:93]: move line93 after line95?

move line99 after line102

and make one extra line between the two blocks?

Comment (Fructokinase) [reply scripts/install_custom_dc.sh]: How does it look now?

**Review by shifucun — APPROVED** [review]
thanks for update!

File: scripts/install_custom_dc.sh (hunk 1 above)
Comment (shifucun) [reply scripts/install_custom_dc.sh]: LG!

File: scripts/install_custom_dc.sh [hunk 2]
```diff
@@ -83,8 +83,32 @@ terraform init && terraform apply \
...
+Please don't forget to email support+custom@datacommons.org with your
+GCP project id for data access.
+###############################################################################
+
+
+###############################################################################
+# Action required:
+Please also make sure to click on the activation email for $DOMAIN
+If the contact email has previously been used to verify domains,
+then $DOMAIN will be already active without needing activation emails.
```
Comment (beets) [inline scripts/install_custom_dc.sh:103]: Is this the same as saying: "This step is only required for new and unverified domains"?
The "Action required" heading is also optional, in this case.

**Pull request merged**